spec/spec.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Opening this HTML file will run the tests inside the browser, instead of in Node JS